class-gwiz-gf-feed-forge.php

```diff
@@ -461,7 +462,23 @@ public static function process_entry_feeds( $entries, $feeds, $form_id ) {
 					continue;
 				}
 
-				$addon = $addons[ $feed['addon_slug'] ];
+				$addon         = $addons[ $feed['addon_slug'] ];
+				$entry         = GFAPI::get_entry( $entry_id );
+				$condition_met = $addon->is_feed_condition_met( $feed, $form, $entry );
+
+				/**
+				 * Filters whether to reprocess feeds that have already been processed.
+				 *
+				 * @param bool $condition_met Whether feed condition is met.
+				 * @param array $feed The current feed.
+				 * @param array $form The current form.
+				 * @param array $entry The current entry.
+				 *
+				 * @since 1.0.2
+				 */
+				if ( ! gf_apply_filters( array( 'gfff_condition_met', $form_id ), $condition_met, $feed, $form, $entry ) ) {
```

Comment:
Can you rename this to `gfff_feed_condition_met` for clarity in snippets? I'd update `$condition_met` to `$feed_condition_met` as well.